f/flatbuffers/flatbuffers_ubi8.5.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package       : flatbuffers
+# Version       : v1.12.0
+# Source repo   : https://github.com/google/flatbuffers.git
+# Tested on     : UBI 8.5
+# Language      : C++
+# Travis-Check  : true
+# Script License: Apache License, Version 2 or later
+# Maintainer    : BulkPackageSearch Automation {maintainer}
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=flatbuffers
+PACKAGE_VERSION=v1.12.0
+PACKAGE_URL=https://github.com/google/flatbuffers.git
+
+yum install -y git wget gcc-c++ cmake
+yum -y update --allowerasing --skip-broken --nobest
```

Comment:
why is this needed?

Reply:
its to resolve version issue with already installed packages.

Reply:
removed this line and the build / travis check passed